string room_id
bool is_completed
bool door_status
geometry_msgs/PoseStamped goal
